Comprehension-2.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Comprehension #2</title>
+</head>
+<body>
+    <h1>Understanding objects in combination with functions</h1>
+    <script>
+        var dog1 = {
+            name: "Barky",
+            species: "dog",
+            breed: "labrador",
+            weight: 20
+        };
+        var dog2 = {
+            name: "Bello",
+            species: "dog",
+            breed: "mixed",
+            weight: 30
+        };
+
+        var cat1 = {
+            name: "Baboo",
+            species: "cat",
+            breed: "ocelot",
+            weight: 60
+        };
+        var cat2 = {
+            name: "Kitty",
+            species: "cat",
+            breed: "egypt",
+            weight: 15
+        };
+        var toy1 = {
+            name: "Furby",
+            species: "toy",
+            breed: "Hasbro",
+            weight: 1
+        };
+        var toy2 = {
+            name: "Pikachu",
+            species: "toy",
+            breed: "Nintendo",
+            weight: 0.5
+        };
+        var cat3 = {
+            name: "Tardar Sauce",
+            species: "cat",
+            breed: "Reddit",
+            weight: 1
+        };
+        var dog3 = {
+            name: "Spunky",
+            species: "dog",
+            breed: "mops",
+            weight: 3
+        };
+
```

Comment:
Do you have an idea to make the objects more compact?

Reply:
there is no real way to make it more compact, but later on you most likely wont hardcode things like that :D

Reply:
Thanks! Now I gained some more self assurance regarding my skills for writing JS Code :)